internal/config/config.go

```diff
@@ -343,6 +343,40 @@ func (c Config) ReadConfig(r io.Reader) (dynOnly bool, err error) {
 	return dynOnly, nil
 }
 
+// RedactSensitiveInfo - removes sensitive information
+// like urls and credentials from the configuration
+func (c Config) RedactSensitiveInfo() {
+	sensitiveConfigKeys := map[string][]string{
+		"notify_amqp":          {"url", "routing_key"},
+		"notify_nsq":           {"nsqd_address"},
+		"notify_postgres":      {"connection_string"},
+		"notify_kafka":         {"sasl_username", "sasl_password", "client_tls_cert", "client_tls_key"},
+		"notify_redis":         {"address", "key", "password"},
+		"notify_mqtt":          {"broker", "password", "username"},
+		"notify_mysql":         {"dsn_string"},
+		"notify_webhook":       {"endpoint", "client_cert", "client_key"},
+		"notify_elasticsearch": {"url", "username", "password"},
+		"notify_nats":          {"address", "username", "password", "token", "cert_authority", "client_cert", "client_key"},
```

Comment:
you should use the relevant constants here @anjalshireesh and enhance the `config.Help` to have a boolean to indicate that if the 'key' is sensitive or not. 

If this set then the code here can simply look through HelpKV and ignore the keys that are present in the config with '*redacted*' value.

Reply:
done